src/plugins/bgimg/bgimg.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @title WET-BOEW Set background image
+ * @overview to be replaced by CSS 4: background-image:attr(data-bgimg, url)
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var $document = wb.doc,
+	componentName = "wb-bgimg",
+	selector = ".provisional[data-bgimg], .provisional [data-bgimg]",
```

Comment:
Need to be also compatible with the deprecated markup that we have introduced in GCWeb. 
We would deprecate the old markup and we would remove it in 6 month. That would require a note in teh GCWeb provisional page.